inst/NEWS.Rd

```diff
@@ -19,6 +22,12 @@
       \item Several vignettes now use the \code{collapse} argument to
       show output in the corresponding code block.
     }
+    \item Changes in Rcpp Deployment:
+    \itemize{
+      \item Dockerfiles for contiunous integration, standard deployment
```

Comment:
typo: contiunous -> continuous

Reply:
Thank you!  `fortunes::fortune(112)` again :)